cloudbuild-main.yaml

```diff
@@ -9,9 +9,7 @@ steps:
   # https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/docker
   # https://cloud.google.com/build/docs/kaniko-cache
   - id: build
-    # Kaniko pinned to earlier version due to
```

Comment:
Is it worth pinning to the latest specific version so that it doesn't update out from under us?

Reply:
I had considered that, but figured it'd be fine to get the benefit of auto update versus the risk of some version breaking us.